recipes/jupyter-rsession-proxy/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "jupyter-rsession-proxy" %}
+{% set version = "1.0b6" %}
```

Comment:
There are no official non-dev releases yet. Once there is a stable version, we keep the conda package at the latest stable.